admin_manual/configuration/server/occ_command.rst

```diff
@@ -61,7 +61,7 @@ For example, in CentOS 6.5 with SCL-PHP54 installed, the command looks like this
 Running ``occ`` with no options lists all commands and options, like this example on Ubuntu::
 
  sudo -u www-data php occ 
- ownCloud version 9.0.0
+ ownCloud version 10.0.8
```

Comment:
No, this is just the general occ command and when typed you see a version string printed. Because v9 is "old" I wanted to update this to a actual release number. I did that on some other places too. Looks now up-to-date :smile: